sky/packages/sky/lib/src/rendering/custom_layout.dart

```diff
@@ -16,41 +18,45 @@ abstract class MultiChildLayoutDelegate {
   /// can reflect that.
   Size getSize(BoxConstraints constraints) => constraints.biggest;
 
+  /// True if a non-null LayoutChild was provided for the specified id.
+  bool isChild(Object childId) => _idToChild[childId] != null;
```

Comment:
I struggled with this name. The delegate doesn't really "have children". Plus "hasChild" might imply that the Widget named childId has-a child.